### DISCUSSION TITLE
Irrelevant

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [community/community#57704] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

The Feed is irrelevant. I don't care what's trending or what you think is "relevant" for me.

Github isn't a social network. I don't care about a feed at all.

**Comment by alcedo** [comment]
Agree - i just want to go back to work on my latest projects, and the TOP repository list is never accurate based on the last few repository that i am working on
